### PR TITLE
My_Portfolio: Milestone 6

### PR DESCRIPTION
**Pull request by Donmark2k**
In this Milestone, We

1. created a new branch, created_menu_feature
2.  update the linters
3. update the readMe file
4. make the hamburger menu to be interactive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,44 +10,35 @@
...
-        <div class="collapse navbar-collapse justify-content-center" id="nav-menu">
-          <ul class="navbar-nav">
-            <li class="nav-item">
-              <a href="#portfolio" class="nav-link">Portfolio</a>
+
+        <!-- Navigation Section  -->
+
+    <header>
+      <nav class="navbar">
+        <a href="" class="nav-branding welcome">Welcome</a>
```
Comment (Davidosky007) [inline index.html:18]: * As shown in the design template kindly hide the welcome text when rendering your drop-down menu 
* ## Given Design 
image
## Yours 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Donmark2k @lorenzoworx 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so-close

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Donmark2k) [reply index.html:18]: solved by setting the display to hidden

**Review by OyePriscilla — APPROVED** [review]
Hi @Donmark2k 👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters check passed ✔️ 
Good commit messages ✔️ 
GitHub flow observed 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**